backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/card/internal/BpkCardImpl.kt

```diff
@@ -41,39 +42,55 @@ import net.skyscanner.backpack.compose.tokens.BpkSpacing
 @Composable
 internal inline fun CardContent(
     padding: BpkCardPadding,
-    contentAlignment: Alignment,
-    content: @Composable BoxScope.() -> Unit,
+    content: @Composable ColumnScope.() -> Unit,
 ) {
-    Box(
+    Column(
         modifier = Modifier.padding(
             all = when (padding) {
                 BpkCardPadding.None -> 0.dp
                 BpkCardPadding.Small -> BpkSpacing.Base
             },
         ),
-        contentAlignment = contentAlignment,
         content = content,
     )
 }
 
 @Composable
-internal fun cardBackgroundColor(elevation: BpkCardElevation): Color =
-    animateColorAsState(
-        when (elevation) {
+internal fun cardColors(elevation: BpkCardElevation): CardColors {
+    val value by animateColorAsState(
+        targetValue = when (elevation) {
             BpkCardElevation.Focus -> BpkTheme.colors.surfaceElevated
             BpkCardElevation.None, BpkCardElevation.Default -> BpkTheme.colors.surfaceDefault
         },
-    ).value
+        label = "BpkCard background color",
+    )
+    return CardDefaults.cardColors(
+        containerColor = value,
+        contentColor = BpkTheme.colors.textPrimary,
+        disabledContainerColor = BpkTheme.colors.surfaceDefault,
```

Comment:
wouldn't `disabledContainerColor` just match `containerColor`?

Reply:
It does match – see the when statement above. The reason why we don't use the value for disabled is that we don't want to use elevated color if the card is disabled and elevated.

Reply:
It feels like quite unexpected behaviour as we've got cards that are on top of `surfaceDefault` and if they're disabled they'll blend into the background. we don't have a disabled style for cards defined and as the text colour matches the enabled one the container colour should too to avoid confusion

Reply:
Perhaps we can remove this parameter in the first place?

Reply:
Done